karma.conf.js

```diff
@@ -72,7 +72,7 @@ module.exports = function(config) {
 
     // level of logging
     // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
-    logLevel: config.LOG_INFO,
+    logLevel: config.LOG_ERROR,
```

Comment:
Probably safe to use LOG_WARN here...it'll provide some more useful output while not being overly verbose.